doc/examples/servinit.cfg

```diff
@@ -31,26 +31,6 @@ if (= $rehashing 0) [
     // serverport 28801 // port which server binds to (you must open this port [UDP] and this plus one, default 28801 and 28802); cmdline: -spN
     // servermasterport 28800 // master server port which server *connects* to; cmdline: -saN
     //
```

Comment:
these are still needed! We just want to remove the client implementation, not the server implementation!